go.mod

```diff
@@ -4,112 +4,112 @@ go 1.20
 
 require (
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.27.4
+	github.com/onsi/gomega v1.27.6
 	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.56.3
 	github.com/prometheus-operator/prometheus-operator/pkg/client v0.56.3
-	github.com/prometheus/client_golang v1.14.0
-	github.com/prometheus/common v0.37.0
-	github.com/spf13/cobra v1.6.0
+	github.com/prometheus/client_golang v1.16.0
+	github.com/prometheus/common v0.44.0
+	github.com/spf13/cobra v1.7.0
 	github.com/stretchr/testify v1.8.4
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.27.2
-	k8s.io/apimachinery v0.27.2
-	k8s.io/apiserver v0.27.2
-	k8s.io/client-go v0.27.2
-	k8s.io/component-base v0.27.2
-	k8s.io/klog/v2 v2.90.1
-	k8s.io/kube-openapi v0.0.0-20230501164219-8b0f38b5fd1f
-	k8s.io/metrics v0.27.2
-	sigs.k8s.io/custom-metrics-apiserver v1.27.0
+	k8s.io/api v0.28.1
+	k8s.io/apimachinery v0.28.1
+	k8s.io/apiserver v0.28.1
+	k8s.io/client-go v0.28.1
+	k8s.io/component-base v0.28.1
+	k8s.io/klog/v2 v2.100.1
+	k8s.io/kube-openapi v0.0.0-20230901164831-6c774f458599
+	k8s.io/metrics v0.28.1
+	sigs.k8s.io/custom-metrics-apiserver v1.28.0
```

Comment:
Need to use latest custom-metrics-apiserver, https://github.com/kubernetes-sigs/prometheus-adapter/pull/655

Reply:
@CatherineF-dev thanks for letting me know, updated the version

Reply:
@CatherineF-dev  @dgrisonnet Could you please help  why verify lint test is failing? I ran the verify lint command on my local and it passes fine

<img width="1282" alt="image" src="https://github.com/kubernetes-sigs/prometheus-adapter/assets/119214363/c31b861b-540a-42f5-99d3-075c89aadd67">
<img width="1579" alt="image" src="https://github.com/kubernetes-sigs/prometheus-adapter/assets/119214363/3dfe72b4-6e79-4930-8944-43ca7a89d237">

Reply:
I have reverted the GOLANGCI_VERSION for now